cmd/egress/cmd.go

```diff
@@ -42,6 +43,13 @@ func NewCmdValidateEgress() *cobra.Command {
 
 	validateEgressCmd := &cobra.Command{
 		Use: "egress",
+		Short: "Verify essential openshift domains are reachable from given subnet ID",
+		Long: dedent.Dedent(`Verify essential openshift domains are reachable from given VPC subnet ID.
+Each subnet must be verified individually.`),
+		Example: dedent.Dedent(`(IMPORTANT: For AWS, ensure your cloud credential environment vars AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are set correctly before execution)
```

Comment:
Also the TOKEN if using a set of STS credentials

Reply:
added